CRM/Contribute/BAO/Contribution.php

```diff
@@ -5457,4 +5457,25 @@ public static function createProportionalFinancialEntries($entityParams, $lineIt
     }
   }
 
+  /**
+   * Calculate Tax for each item when Financial Type is changed.
+   *
```

Comment:
It would be nice to add the logic of the calculation to the method description. Otherwise, we have to read the code in order to understand what is going on